data/json/items/tools.json

```diff
@@ -8272,7 +8277,7 @@
         "moves": 30
       }
     ],
-    "flags": [ "FIRE", "LIGHT_240", "CHARGEDIM", "FLAMING", "DURABLE_MELEE", "TRADER_AVOID" ]
+    "flags": [ "FIRE", "LIGHT_240", "CHARGEDIM", "FLAMING", "DURABLE_MELEE", "TRADER_AVOID", "FIRESTARTER" ]
```

Comment:
Both `"FIRE"` and `"LIGHT_240"` are presumably dead flags too.

Reply:
I'm not sure about `LIGHT_X` - see https://github.com/CleverRaven/Cataclysm-DDA/blob/master/src/item_factory.cpp#L165-L168.

Reply:
`FIRE` flag is dead for items - I've removed it in #22699

Reply:
On the other hand, `FIRE` flag is dead only in current master, but in this PR it defines item which is already burning, so it can be used to ignite something without spending any additional charges.

Reply:
> I'm not sure about LIGHT_X

You're right, it's still used (though a bit awkwardly). Please, disregard my comment concerning it.

> in this PR it defines item which is already burning, so it can be used to ignite something without spending any additional charges.

Which means they're mutually exclusive and mustn't appear together in one item.

Reply:
> Which means they're mutually exclusive and mustn't appear together in one item.

Thanks, I've remove `FIRESTARTER` flag from those items which already have `FIRE` flag.